data/presets/baking_oven/bakehouse.json

```
@@ -0,0 +1,24 @@
+{
+    "icon": "Bakehouse",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "amenity": "baking_oven",
+        "cuisine": "Bakehouse"
+    },
+    "reference": {
+        "key": "cuisine"
+    },
+    "terms": [
+        "breads",
+        "croissants",
+        "muffins",
+        "tea Cakes",
+        "bagels",
+        "nibbles",
+        "cookies"
```

are you sure about the _pastry_ type products as search terms for this preset? I think that typically, such baking ovens are used to only bake breads and bread-like products.

Suggestion:

```suggestion
        "beehive oven",
        "bread",
        "clay oven",
        "horno",
        "masonry oven",
        "tabun oven",
        "pizza oven"
```

---

Thank you so much for the suggestion implemented the same....